pom.xml

```diff
@@ -127,30 +132,56 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
                 <executions>
+                    <!-- prepare JaCoCo agent for surefire:test@unit-test -->
                     <execution>
-                        <id>default-prepare-agent</id>
+                        <id>unit-prepare-agent</id>
                         <goals>
                             <goal>prepare-agent</goal>
                         </goals>
-<!--                        <configuration>-->
-<!--                            <destFile>${project.build.directory}/jacoco-default.exec</destFile>-->
-<!--                        </configuration>-->
+                        <phase>initialize</phase>
+                        <configuration>
+                            <propertyName>jacoco-agent.unit-tests</propertyName>
+                            <destFile>${project.build.directory}/jacoco-unit-tests.exec</destFile>
```

Comment:
💬 Give a descriptive name for the unit tests report. This value will also land in the property above.